src/apprt/gtk/Window.zig

```diff
@@ -88,6 +91,7 @@ pub const DerivedConfig = struct {
             .gtk_wide_tabs = config.@"gtk-wide-tabs",
             .gtk_toolbar_style = config.@"gtk-toolbar-style",
 
+            .title = config.title,
```

Comment:
This should be copied, and then freed in a `deinit` function, or when reconfigured.